lib/modules.nix

```diff
@@ -156,7 +156,10 @@ rec {
             type = types.lazyAttrsOf types.raw;
             # Only render documentation once at the root of the option tree,
             # not for all individual submodules.
-            internal = prefix != [];
+            # Allow other mkOption decls to make this internal regardless.
+            ${if prefix == []
+              then null # unset; default is true
```

Comment:
I'm pretty sure the default is `false`?

Reply:
Right, I must have been thinking of visibility instead of internalness. Now it's:

    then null  # unset => visible